.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Thank you for submiting a contribution to Apache NiFi.
+
+In order to streamline the review of the contribution we ask you
+to ensure the follow steps have been taken:
+
+### For all changes:
+- [] Is there a JIRA ticket associated with this PR? Is is referenced 
```

Comment:
We should additionally request that they get the format of the title right as GitHub is wont to strip out the hyphen just for the sake of consistency.